management/server/network.go

```diff
@@ -56,13 +62,31 @@ func NewNetwork() *Network {
 	r := rand.New(s)
 	intn := r.Intn(len(sub))
 
+	var n6 *net.IPNet = nil
+	if enableV6 {
+		n6 = GenerateNetwork6()
+	}
+
 	return &Network{
 		Identifier: xid.New().String(),
 		Net:        sub[intn].IPNet,
+		Net6:       n6,
 		Dns:        "",
 		Serial:     0}
 }
 
+func GenerateNetwork6() *net.IPNet {
```

Comment:
This seems to generate the same range on every execution.

Can you use the Read function from the crypto/rand package?

Also, can you use the range `fd00:b14d:RAND_BYTES::/64` ? Where `b14d` = `bird`

Reply:
This will make the ranges easy to identify and possibly memorize (not that most people wants that)

Reply:
> This seems to generate the same range on every execution.
> 
> Can you use the Read function from the crypto/rand package?

I have replaced the function call-specific RNG for all IP address generation functions with a global one that is seeded based on `"crypto/rand".Read()`.
With this change, the IP addresses and network prefixes are now different on every call.

> Also, can you use the range `fd00:b14d:RAND_BYTES::/64` ? Where `b14d` = `bird`

Done.

Reply:
thanks